lstchain/reco/utils.py

```diff
@@ -649,12 +699,12 @@ def get_effective_time(events):
     t_eff: float
     t_elapsed: float
     """
-    
-    delta_t = events.delta_t[1:]
+
+    delta_t = events["delta_t"][1:]
```

Comment:
I had forgotten to mention this again in the earlier PR, but made the small change here, so that the events can be read in QTable format as well

Reply:
> I had forgotten to mention this again in the earlier PR, but made the small change here, so that the events can be read in QTable format as well

Hi, you can now drop those changes, which conflict with the ones in #615

Reply:
Apart from fixing the conflict there is nothing else left, right? 

To those who provided feedback: please approve, we can address possible issues (that may come up when applying this to more data) after the coming release.

Reply:
Yes, I will make the new commit soon. Beyond this, I do not think there is anything urgent for this PR.

Reply:
I have a minor issue that I overlooked:
```
time_diff = np.diff(events['dragon_time']) 
t_elapsed = np.sum(time_diff[time_diff<0.1])
```
The comparison in the second line won't work when using QTables, as `time_diff` is an `astropy.units.Quantity` object.
I am checking if there is a way to make it work for both when the events is a `pandas.DataFrame` or `astropy.table.QTable` object.
But if @maxnoe or anyone else has a quicker fix, let me know

Reply:
> I have a minor issue that I overlooked:
> 
> ```
> time_diff = np.diff(events['dragon_time']) 
> t_elapsed = np.sum(time_diff[time_diff<0.1])
> ```
> 
> The comparison in the second line won't work when using QTables, as `time_diff` is an `astropy.units.Quantity` object.
> I am checking if there is a way to make it work for both when the events is a `pandas.DataFrame` or `astropy.table.QTable` object.
> But if @maxnoe or anyone else has a quicker fix, let me know

Let me fix this in another PR

Reply:
> > I have a minor issue that I overlooked:
> > ```
> > time_diff = np.diff(events['dragon_time']) 
> > t_elapsed = np.sum(time_diff[time_diff<0.1])
> > ```
> > 
> > 
> > The comparison in the second line won't work when using QTables, as `time_diff` is an `astropy.units.Quantity` object.
> > I am checking if there is a way to make it work for both when the events is a `pandas.DataFrame` or `astropy.table.QTable` object.
> > But if @maxnoe or anyone else has a quicker fix, let me know
> 
> Let me fix this in another PR

I can try to edit the code in `lstchain.io.read_data_dl2_to_QTable` so that it returns effective and elapsed time as well as the modified QTables object.
I don't seem to find an easier solution

Reply:
With that, we won't have to bother about this issue in the main `get_effective_time` function.

Reply:
Let me have a look. We can just check if the things are quantities with units, and act accordingly. We stick to returning the times as values with no units, right?

Reply:
I do not think we will have an issue if the values returned have units or not

Reply:
Please check #617.
In the end I did the opposite, I return times in seconds always, which seems nicer to me. 
@misabelber @chaimain  : if this change of the returns breaks something I can go back, though.